gatling-jms/src/main/scala/io/gatling/jms/action/JmsRequestTrackerActor.scala

```diff
@@ -74,6 +74,7 @@ class JmsRequestTrackerActor(statsEngine: StatsEngine) extends BaseActor {
 
   private val sentMessages = mutable.HashMap.empty[MessageKey, MessageSent]
   private val receivedMessages = mutable.HashMap.empty[MessageKey, MessageReceived]
+  private val historicallySentMessages = mutable.HashMap.empty[MessageKey, Null]
```

Comment:
* This can grow infinitely. Please use a LinkedHashMap based LRU where you use insert order and override `removeEldestEntry`. Limit should be configurable from `gatling.conf`
* Please don't use null as  values. Use Boolean.TRUE instead.